.github/workflows/ci.yml

```
@@ -42,17 +42,16 @@ jobs:
       matrix:
         include:
           # Linux, gcc
-          - { compiler: gcc-5, cxxstd: '14',       boostBranch: boost-1.65.0, os: ubuntu-22.04, container: 'ubuntu:18.04' }
-          - { compiler: gcc-5, cxxstd: '14',       boostBranch: boost-1.77.0, os: ubuntu-22.04, container: 'ubuntu:18.04' }
-          - { compiler: gcc-12,cxxstd: '14,17,20', boostBranch: master,       os: ubuntu-22.04 }
-          
+          - { compiler: gcc-8, cxxstd: '14,17,20', boostBranch: boost-1.86.0, os: ubuntu-20.04 }
```

How about:
- Oldest: 7
- Oldest with C++20 support: 10 (earlier hat 2a)
- (current) latest: 14 and slightly older 12 with boost-master

- ```suggestion
          - { compiler: gcc-7, cxxstd: '14,17',    boostBranch: boost-1.86.0, os: ubuntu-20.04 }
          - { compiler: gcc-10,cxxstd: '14,17,20', boostBranch: boost-1.86.0, os: ubuntu-22.04 }
          - { compiler: gcc-12,cxxstd: '14,17,20', boostBranch: master,       os: ubuntu-22.04 }
```

---

Sure!